### DISCUSSION TITLE
PR for 22' grads is taking to long to be merged

### DISCUSSION THREAD

**Discussion by Omar-Khas** [community/community#17717] [GitHub Education]
It's been 2hrs since my PR changes have been approved but until now none of the reviewers have checked if I'm compliant with the COC. I've checked other people's PRs and they are being merged after an hour or less. OtherPRs have been merged although they've been created later than mine. Anyone knows if there's something I can or should do other than waiting? Here's a link to my PR:
[https://github.com/education/GitHubGraduation-2022/pull/6193#pullrequestreview-980811134](url)

**Comment by junaydirfan** [comment]
Same here, mate

**Comment by mlarasusan** [comment]
Hello [Omar-Khas](https://github.com/Omar-Khas), so sorry about that, we're getting a high volume of applications. Looks like your PR is merged. Yours as well [junaydirfan](https://github.com/junaydirfan). Congratulations!
